apps/web/components/booking/BookingListItem.tsx

```diff
@@ -353,6 +354,25 @@ function BookingListItem(booking: BookingItemProps) {
                   attendees={booking.attendees}
                 />
               </div>
+              {booking.location && (
+                <Link href={bookingLink} className="text-sm leading-6 text-blue-400 hover:underline">
+                  <div className="flex items-center gap-2">
+                    {eventLocationType ? (
+                      <>
+                        <img
+                          src={eventLocationType.iconUrl}
+                          className="h-4 w-4 rounded-sm"
+                          alt={`${eventLocationType.label} logo`}
+                        />
+                        Join {eventLocationType.label}
```

Comment:
can you add this to en/common.json and use `t("join", eventLocationType.label)` ?